src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5399,10 +5399,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>warning action enable</value>
   </data>
   <data name="WRN_ConvertingNullableToNonNullable" xml:space="preserve">
-    <value>Converting null literal or possible null value to non-nullable type.</value>
+    <value>Null literal or possibly null value is being converted to a non-nullable type.</value>
```

Comment:
I have two concerns regarding this proposed change:

1. I don't find the new wording significantly more readable.
1. We have a few different ways similar messages are presented in error messages. I could not find any which have this sentence structure, meaning this may be increasing overall confusion in messaging by adding another form.

My preference is to leave this unchanged, but otherwise I would offer the following as a secondary option:

> Conversion of null literal or possible null value to non-nullable type

Reply:
> I have two concerns regarding this proposed change:
> 
> 1. I don't find the new wording significantly more readable.

It is not supposed to be more readable.  It is supposed to be less confusing to the translator.

> 2. We have a few different ways similar messages are presented in error messages. I could not find any which have this sentence structure, meaning this may be increasing overall confusion in messaging by adding another form.

OTOH, having this thing in the subject significantly reduces the chance of botching the translation.

> 
> My preference is to leave this unchanged, but otherwise I would offer the following as a secondary option:
> 
> > Conversion of null literal or possible null value to non-nullable type

This one does not fix the problem I am fighting at all.

Reply:
> It is not supposed to be more readable. It is supposed to be less confusing to the translator.

This is not the manner in which we convey additional information to the translator. The _sole purpose_ of this value is to provide value to English-reading users, and this is the only metric by which a change to this line will be evaluated. We will provide additional context to the translators separately based on instructions from localization coordinators.

Reply:
> It is not supposed to be more readable. It is supposed to be less confusing to the translator.

There are ways we already have to pass information to a translator to make things less confusing. 

For example, we can add a `<comment>` tag to add context here. :-)